### PR TITLE
ignore cluster fields in checksum, to avoid frequent status changes

### PR DESCRIPTION
**Pull request by rccrdpccl**
the aim of this PR is to remove some fields from cluster metadata when determining the checksum. This will help us reducing the number of cluster events due to unnecessary fields

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -34,6 +34,8 @@ objects:
             - secretRef:
                 name: events-scrape
           env:
+          - name: EVENT_STORE_CLUSTER_EVENTS_IGNORE_FIELDS
+            value: "hosts[].connectivity,hosts[].progress"
```
Comment (rccrdpccl) [inline openshift/template.yaml:38]: We will most likely add more

File: assisted-events-scrape/dictpath/dictpath.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+from typing import List
```
Comment (osherdp) [inline assisted-events-scrape/dictpath/dictpath.py:1]: It's very similar to https://pypi.org/project/path-dict/ no?
If so, I prefer using that external project (or anything alike)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: assisted-events-scrape/dictpath/dictpath.py (hunk 2 above)
Comment (osherdp) [reply assisted-events-scrape/dictpath/dictpath.py:1]: There is https://github.com/dpath-maintainers/dpath-python
Just reading a bit through it, it seems to support lists and filtering/deletion

Comment (rccrdpccl) [reply assisted-events-scrape/dictpath/dictpath.py:1]: I think this one would do, thanks for finding it!! I'll update this PR

**Comment by rccrdpccl** [comment]
Integration test keeps failing due to cluster claim timing out, I'm looking into it but if it keeps timing out I will override it once all other changes look good

**Review by osherdp — APPROVED** [review]

File: assisted-events-scrape/dictpath/dictpath.py (hunk 2 above)
Comment (osherdp) [reply assisted-events-scrape/dictpath/dictpath.py]: Yeah looks great
Thanks!

**Pull request merged**